### PR TITLE
Referencing env vars in configs

### PR DESCRIPTION
**Pull request by falnyr**
As mentioned here: https://stackoverflow.com/a/53459699/2111274

Thanks for contributing, you're awesome! :star:
Please describe expected and observed behaviour, and what you're fixing.
For visual fixes, please include tested browsers and screenshots.
Search for related existing issues and link to them if possible.
Please read https://docs.silverstripe.org/en/contributing/code/

### CODE REVIEW

File: docs/en/00_Getting_Started/03_Environment_Management.md [hunk 1]
```diff
@@ -48,6 +48,17 @@ use SilverStripe\Core\Environment;
...
+### Using environment variables in config
+
+To use environment variables in `.yaml` configs you can reference them using backticks.
+
+```yaml
+SilverStripe\Core\Injector\Injector:
+  MyServiceClass:
+    properties:
+      MyProperty: '`ENV_VAR_HERE`'
+```
```
Comment (robbieaverill) [inline docs/en/00_Getting_Started/03_Environment_Management.md]: It might be worth noting that this only works for injector configuration

**Comment by ScopeyNZ** [comment]
I wonder if this is better suited in the [configuration docs](https://docs.silverstripe.org/en/4/developer_guides/configuration/). I am a little surprised it's missing.

**Comment by falnyr** [comment]
> I wonder if this is better suited in the [configuration docs](https://docs.silverstripe.org/en/4/developer_guides/configuration/). I am a little surprised it's missing.

I think it can be in both (mainly here). Usually I look for environment variables and how I can use them in a framework without needing to know there is a "Configuration API".

**Comment by robbieaverill** [comment]
I agree

**Comment by ScopeyNZ** [comment]
Thanks @falnyr ! The change should propagate to the docs over the next few days

**Pull request merged**